src/main/c/netty_io_uring_native.c

```diff
@@ -218,10 +218,13 @@ static jboolean netty_io_uring_probe(JNIEnv *env, jclass clazz, jint ring_fd, ji
     return supported;
 }
 
-
-static jobjectArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries) {
-    struct io_uring_params p;
-    memset(&p, 0, sizeof(p));
+static jobjectArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries, jint wqFd) {
+    struct io_uring_params params;
+    memset(&params, 0, sizeof(params));
+    if (wqFd > 0) {
```

Comment:
should we better use > -1 and use -1 if we don't want to use a wqFd. ?

Reply:
Actually it seems like we use -1 in the java code already. So I think just this if branch should be changed